src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -332,13 +333,15 @@ class DiskImageEditor extends Component {
                 }
                 { !createMode && msg.diskEditorResizeLabel() }
```

Comment:
I'm thinking that instead of an onHover Tooltip on the field, displaying the message via a FieldLevelHelp popover would be more consistent with the dialog.  So really it would be just copying the style above and using the new tooltip message as the help's content.